extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 		$this->response->setVal( 'status', $status );
 
 		if ( $status ) {
-			wfRunHooks( 'CuratedContentSave' );
+			wfRunHooks( 'CuratedContentSave', [ $sections ] );
```

Comment:
@Warkot We're purging our internal request and all actions on it - i.e. parsing and formatting data.